### ISSUE TITLE
Can we callout the setup.particle.io link more clearly?

### ISSUE THREAD

**Issue by dmiddlecamp** [particle-iot/docs#673]
it's hard to spot this as important in a sea of text:

page: https://docs.particle.io/guide/getting-started/connect/electron/ 

<img width="1243" alt="screen shot 2017-08-08 at 9 22 05 am" src="https://user-images.githubusercontent.com/833793/29076417-1b2c745a-7c1b-11e7-8b49-2cff16b7a168.png">

**Comment by QuadeHale** [comment]
I agree. I received my Electron and spent more time figuring out how to add it to my account than I have writing code.

**Comment by ScruffR** [comment]
>  I received my Electron and spent more time figuring out how to add it to my account than I have writing code.

Did you not get one of these cards with your devices?
![win_20180526_13_57_00_pro](https://user-images.githubusercontent.com/6604878/40575869-570a1fc4-60ed-11e8-8f5d-5c4bb4d71463.jpg)

**Comment by technobly** [comment]
When you drop into the docs the first page has a nice large button that could be reused. 

Also the electron packaging should have a bright cyan cardboard band that surrounds the contents of the kit that has the particle.io/setup link on it.

**Comment by QuadeHale** [comment]
I have the cyan band. It does not mention the URL anywhere.
I did not get the card inside with particle.io/setup.
I did, however, get everything else.
I purchased my Electron through Amazon. I can't find a date on any of this stuff either, so maybe I just got old stock? 
In any case - if the box is supposed to contain all of these things and I got a freak package, then there are some solid odds that someone else has, too. There's no harm in adding it to the docs.
